Doc/c-api/structures.rst

```diff
@@ -399,6 +399,23 @@ definition with the same method name.
    slot.  This is helpful because calls to PyCFunctions are optimized more
    than wrapper object calls.
 
+.. c:function:: PyObject * PyCFunction_New(PyMethodDef *ml, PyObject *self)
+
+   Turn ``ml`` into a Python callable object.  The ``self`` parameter will be
```

Comment:
`ml` and `self` should be marked up in italic (see the devguide's style guide section for details). Please apply to the whole PR.

```suggestion
   Turn *ml* into a Python callable object.  The *self* parameter will be
```